src/Card/Card.scss

```diff
@@ -53,3 +53,173 @@
     flex-shrink: 0;
   }
 }
+
+.pgn__card-divider {
+  border-top: 1px solid $card-divider-bg;
+  height: 0;
+}
+
+.pgn__card-section {
+  padding: $card-spacer-x;
+  flex-grow: 1;
+
+  &.is-muted {
+    background-color: $light-200;
+  }
+
+  .pgn__card-section-title {
+    color: $black;
+    font-weight: $font-weight-bold;
+    font-size: $h5-font-size;
+    margin-bottom: $card-spacer-y;
+  }
+
+  .pgn__card-section-actions {
+    display: flex;
+    justify-content: flex-end;
+    margin-top: $card-spacer-y;
+  }
+}
+
+.pgn__card-footer {
+  width: 100%;
+  padding: $card-spacer-x;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &.vertical {
+    justify-content: flex-end;
+    padding-top: 0;
+
+    & > * + * {
+      margin-left: $card-footer-actions-gap;
+    }
+  }
+
+  &.vertical-stacked {
+    flex-direction: column-reverse;
+    padding-top: $card-spacer-x - $card-footer-actions-gap;
+
+    & > * {
+      width: 100%;
+      margin-top: $card-footer-actions-gap;
+    }
+  }
+
+  &.horizontal {
+    flex-wrap: wrap;
+
+    & > :not(:last-child) {
+      margin-right: $card-footer-actions-gap;
+    }
+  }
+
+  &.horizontal-stacked {
+    flex-direction: column-reverse;
+    padding-top: $card-spacer-x - $card-footer-actions-gap;
+
+    & > * {
+      margin-top: $card-footer-actions-gap;
+    }
+  }
+}
+
+.pgn__card-footer-text {
+  color: $gray-700;
+  order: 1;
+
+  &.vertical {
+    margin-right: auto;
+    order: -1;
+  }
+
+  &.vertical-stacked {
+    margin-right: auto;
+  }
+
+  &.horizontal {
+    margin-top: $card-spacer-x;
+    width: 100%;
+    text-align: center;
+  }
+
+  &.horizontal-stacked {
+    margin-top: $card-spacer-x;
+    width: 100%;
+    text-align: center;
+    order: -1;
+  }
+}
+
+.pgn__card-wrapper-image-cap {
+  position: relative;
+
+  &.horizontal {
+    max-width: 30%;
+
+    .pgn__card-image-cap {
+      height: 100%;
+      border-top-left-radius: $card-image-border-radius;
+      border-bottom-left-radius: $card-image-border-radius;
+      border-top-right-radius: 0;
+      max-width: -moz-available;          /* WebKit-based browsers will ignore this. */
+      max-width: -webkit-fill-available;  /* Mozilla-based browsers will ignore this. */
```

Comment:
nit: Are these vendor prefixes necessary, or will the autoprefixer during Webpack build add these for us?

Reply:
Yeah, I've just checked and they will be added automatically, removed these lines, thanks!